Get-VmStatus.ps1

```diff
@@ -0,0 +1,28 @@
+﻿param
+(
+    [Parameter(Mandatory=$false, HelpMessage="Configuration File, see example in directory")]
```

Comment:
New script to print the status of the VMs out to the console for  the labs in the CSV file.  Includes the artifact status as well!